#! /bin/sh /usr/share/dpatch/dpatch-run
## build_wo_debug_info.dpatch by Stefano Zacchiroli <zack@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Avoid building with debugging info ocamldoc and camlp4.
## DP: Wont be needed for the stable release of ocaml 3.10.0.

@DPATCH@
diff -urNad ocaml-3.10.0~beta~/_tags ocaml-3.10.0~beta/_tags
--- ocaml-3.10.0~beta~/_tags	2007-03-04 12:00:26.000000000 +0100
+++ ocaml-3.10.0~beta/_tags	2007-04-16 15:02:20.000000000 +0200
@@ -12,6 +12,8 @@
 
 # We want -g everywhere it's possible
 true: debug
+<ocamldoc/**>: -debug
+<camlp4/**>: -debug
 
 # By default everything we link needs the stdlib
 true: use_stdlib
